test/ng/compileSpec.js

```diff
@@ -7245,22 +7245,22 @@ describe('$compile', function() {
         });
 
         inject(function($compile, $rootScope) {
-          expect(jqLiteCacheSize()).toEqual(0);
+          var cacheSize = jqLiteCacheSize();
```

Comment:
This and a few other tests assume that the cacheSize is 0 after bootstrap. The listener for visibilitychange on the document adds a cache entry. So I thought it makes more sense to not assume a specific cacheSize on bootstrap.